scripts/tests/test_pandas_web.py

```diff
@@ -0,0 +1,65 @@
+from unittest.mock import (
```

Comment:
Feels like this file was committed by mistake as it's a duplicate of the other test? Or am I missing something?

Reply:
Yeah it is a duplicate. I added that because the issue description says the test should be in `scripts/tests/`. I thought the purpose is to make the overall command `python -m pytest scripts/` to cover the test for the web. Please let me know if it is unnecessary.